src/runtime/runtime.js

```diff
@@ -775,10 +775,6 @@ export class ConfiguredRuntime {
 
   /** @override */
   showUpdateOffers(options) {
-    assert(
```

Comment:
Should we ramp the experiment to 100% first? Or instruct the publisher to override with query param (if all they're doing is testing)?

Reply:
@jpettitt I think?

Reply:
After talking to Angie and John we've decided to ramp the experiment instead. Keeping this PR open for the time being, but not submitting.